src/ontology/mondo-edit.obo

```diff
@@ -520167,6 +520139,52 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
 relationship: has_characteristic MONDO:0021152 {source="OMIMPS:616033"} ! inherited
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6777" xsd:anyURI
 
+[Term]
+id: MONDO:0800451
+name: congenital amegakaryocytic thrombocytopenia
+subset: rare
+synonym: "congenital amegakaryocytic thrombocytopenia" EXACT CLINGEN_PREFERRED [OMIMPS:604498]
+xref: OMIMPS:604498 {source="MONDO:equivalentTo"}
+is_a: MONDO:0100241 {source="DOID:0090118", source="MESH:C535982", source="MONDO:Redundant"} ! inherited thrombocytopenia
+relationship: excluded_subClassOf MONDO:0001713 {source="Orphanet:3319", source="https://orcid.org/0000-0001-5208-3432"} ! inherited aplastic anemia
+relationship: has_characteristic MONDO:0021152 {source="OMIMPS:604498"} ! inherited
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6622" xsd:anyURI
+property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/640/congenital-amegakaryocytic-thrombocytopenia" xsd:anyURI {source="GARD:0000640"}
+
+[Term]
+id: MONDO:0800452
+name: congenital amegakaryocytic thrombocytopenia 1
+def: "A rare inherited bone marrow failure syndrome, in which the cause of the disease is a variation in the MPL gene. It is characterized by an isolated and severe decrease in the number of platelets and megakaryocytes during the first years of life that develops into bone marrow failure with pancytopenia later in childhood." [https://orcid.org/0000-0001-5208-3432, Orphanet:3319]
```

Comment:
should Orphanet:3319 be removed since that refers to congenital amegakaryocytic thrombocytopenia vs. congenital amegakaryocytic thrombocytopenia 1?

Reply:
the [Orphanet ID](https://www.orpha.net/consor/www/cgi-bin/OC_Exp.php?Expert=3319) xrefs [OMIM:604498](https://omim.org/entry/604498), so I think it is correct to have it on this term